_source/_includes/tracing-shipping/otel-troubleshooting.md

```diff
@@ -0,0 +1,171 @@
+This section contains some guidelines for handling errors that you may encounter when trying to collect traces with OpenTelemetry.
+
+## Problem: No traces are sent
+
+The code has been instrumented, but the traces are not being sent.
+
+### Possible cause - Collector not installed
+
+The OpenTelemetry collector may not be installed on your system.
+
+#### Suggested remedy
+
+Check if you have an OpenTelemetry collector installed and configured to receive traces from your hosts.
+
+
+### Possible cause - Collector path not configured
+
+If the collector is installed, it may not have the correct endpoint configured for the receiver.
+
+#### Suggested remedy
+
+1. Check that the configuration file of the collector lists the following endpoints:
+
+   ```yaml
+   receivers:
+     jaeger:
+       protocols:
+         thrift_compact:
+           endpoint: "0.0.0.0:6831"
+         thrift_binary:
+           endpoint: "0.0.0.0:6832"
+         grpc:
+           endpoint: "0.0.0.0:14250"
+         thrift_http:
+           endpoint: "0.0.0.0:14268"
+     opencensus:
+       endpoint: "0.0.0.0:55678"
+     otlp:
+       protocols:
+         grpc:
+           endpoint: "0.0.0.0:4317"
+         http:
+           endpoint: "0.0.0.0:4318"
+     zipkin:
+       endpoint: "0.0.0.0:9411"
+   ```
+
+2. In the instrumentation code, make sure that the endpoint is specified correctly. Refer to our [tracing documentation](https://app.logz.io/#/dashboard/send-your-data/collection?tag=all&collection=tracing-sources) for more on this.
+
+
+### Possible cause - Traces not genereated
+
+If the collector is installed and the endpoints are properly configured, the instrumentation code may be incorrect.
+
+
+#### Suggested remedy
+
+
+1. Check if the instrumentation can output traces to a console exporter.
+2. Use a web-hook to check if the traces are going to the output.
+3. Use the metrics endpoint of the collector (http://localhost:8888/metrics) to see the number of spans received per receiver and the number of spans sent to the Logz.io exporter.
+
+If the above steps do not work, refer to our [tracing documentation](https://app.logz.io/#/dashboard/send-your-data/collection?tag=all&collection=tracing-sources) and re-instrument the application.
+
+
+### Possible cause - Wrong exporter/protocol/endpoint
+
+If traces are generated but not send, the collector may be using incorrect exporter, protocol and/or endpoint.
+
+#### Suggested remedy
+
+1. Activate `debug` logs in the configuration file of the collector as follows:
+
+   ```yaml
+   service:
+     telemetry:
+       logs:
+         level: "debug"
+   ```
+
+If you don't see this log (true for opentelemetry-collector version v0.52.0):
+
+```shell
+2022-05-02T13:21:27.973Z	debug	logzioexporter@v0.48.0/y trces logger.go:54	logziosender.go: Sending bulk of 496 bytes
```

Comment:
This log has been deprecated since 0.55.5 and also does not exist in our distro, however, all versions have debug logs that indicate the status code of the http/https post request, So I think we need to have more general suggestion to check the status code in the debug logs and not a specific log format